classes/Cart.php

```diff
@@ -1712,7 +1712,7 @@ public function deleteProduct(
                 WHERE `id_cart` = ' . (int) $this->id . '
                 AND `id_product` = ' . (int) $id_product . '
                 AND `id_customization` = ' . (int) $id_customization .
-                ($id_product_attribute != null ? ' AND `id_product_attribute` = ' . (int) $id_product_attribute : '')
+                (Int) $id_product_attribute ? ' AND `id_product_attribute` = ' . (int) $id_product_attribute : '')
```

Comment:
thank you for your review @psandmore 

Could you please put the I of your "Int" in minuscule? Then I'll approve your pull request, thanx !

Reply:
updated as requested